site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -66,6 +66,47 @@ To install `gcloud` and `kubectl`, perform the following steps:
 
 [gcloud-install]: https://cloud.google.com/sdk/docs/quickstarts
 
+### Choosing a GKE cluster mode
+
+A [cluster][cluster] consists of at least one *control plane* machine and multiple worker machines called *nodes*. In Google Kubernetes Engine, nodes are [Compute Engine virtual machine][vms] instances that run the Kubernetes processes necessary to make them part of the cluster.
+
+Agones supports GKE Standard mode and has alpha support for GKE Autopilot mode.
+
+#### Agones on GKE Autopilot
+
+Autopilot is GKE's fully-managed mode. GKE configures, maintains, scales, and
+upgrades nodes for you, which can reduce your maintenance and operating
+overhead. You only pay for the resources requested by your running Pods, and
+you don't pay for unused node capacity or Kubernetes system workloads.
+
+This section describes the Agones-specific considerations in Autopilot
+clusters. For a general comparison between Autopilot and Standard, refer to
+[Choose a GKE mode of operation](https://cloud.google.com/kubernetes-engine/docs/concepts/choose-cluster-mode).
+
+Autopilot nodes are, by default, optimized for most workloads. If you have
+specific hardware needs such as Arm architecture, specific CPU platforms, or
+hyperthreading disabled, you can specify
+[compute classes](https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-compute-classes)
+for those workloads.
+
+Autopilot has pre-configured opinionated constraints. You might want to use GKE
+Standard mode if these constraints aren't right for you:
+
+*  **Operating system:** No Windows containers.
+*  **Resource requests:** Pre-determined minimum Pod resource requests. If your
+   game servers require less than those minimums, use GKE Standard.
+*  **Scheduling policy:** `Packed` is supported, which is the Agones default.
+   `Distributed` is not supported.
+*  **Host port policy:** `Dynamic` is supported, which is the Agones default.
+   `Static` and `Passthrough` are not supported.
+*  **Seccomp profile:** The `RuntimeDefault` seccomp profile is always enabled.
```

Comment:
This is the opposite of what the UG was trying to say. :) By default, Autopilot uses `RuntimeDefault`, but on Agones, to avoid the issue mentioned (container creation delays), `seccomp` is set to `Unconfined` (or whatever it's called).

Reply:
Changed, but I didnt realize that you could override the seccomp profile in AP